core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -763,21 +758,20 @@ public synchronized void startInternal() throws IOException {
         )
         .collect(Collectors.toSet());
     mRaftGroup = RaftGroup.valueOf(RAFT_GROUP_ID, peers);
-    initServer();
-    super.registerMetrics();
     LOG.info("Starting Raft journal system. Cluster addresses: {}. Local address: {}",
         mClusterAddresses, mLocalAddress);
-    long startTime = System.currentTimeMillis();
     try {
+      initServer();
+      long startTime = System.currentTimeMillis();
       mServer.start();
+      LOG.info("Started Raft Journal System in {}ms", System.currentTimeMillis() - startTime);
     } catch (IOException e) {
       String errorMessage =
           MessageFormat.format("Failed to bootstrap raft cluster with addresses {0}: {1}",
               Arrays.toString(mClusterAddresses.toArray()),
               e.getCause() == null ? e : e.getCause().toString());
-      throw new IOException(errorMessage, e.getCause());
+      throw new RuntimeException(errorMessage, e.getCause());
```

Comment:
just an early comment since it's still in draft, do you think we can use AlluxioRuntimeException here?

Reply:
Yes definitely, thanks for the heads up. I've updated it to use `AlluxioRuntimeException`, let me know if it looks good.